src/index.html

```diff
@@ -5,8 +5,303 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__menu">
+        <ul class="menu">
+          <li class="menu__item "><a href="#" class="menu__link is-active">Apple</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Samsung</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Смартфоны и телефоны</a></li>
+          <li class="menu__item"><a href="#" class="menu__link" data-qa="nav-hover">Ноутбуки и компьютеры</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Гаджеты</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Планшеты</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Фото</a></li>
+          <li class="menu__item"><a href="#" class="menu__link">Видео</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="catalog">
+        <div class="catalog__card" data-qa="card">
+          <a class="catalog__picture">
+            <img src="images/imac.jpeg" alt="imac" class="catalog__img">
```

Comment:
```suggestion
            <img src="images/imac.jpeg" alt="imac" class="card__image">
```
эти элементы первостепенно относятся к карточке, а не к каталогу